lib/logmsg/logmsg.c

```diff
@@ -1852,6 +1852,8 @@ log_msg_lookup_time_stamp_name(const gchar *name)
 
 gssize log_msg_get_size(LogMessage *self)
 {
+  if(!self) return 0;
```

Comment:
I would prefer an assert here.

Reply:
I agree with your reason that a NULL is not a valid LogMessage, but I would not change it.
Using assert would lead an immediate crash which would be unnecessary if we can handle it gracefully.
* There is no valid LogMessage that would be evaluated to 0, therefore a 0 is always an error.
* If we would like to avoid syslog-ng crashes and assert would be preferred then each `log_msg_get_size` call should be guarded. Of course with returning 0 then the user of `log_msg_get_size` should make the error handling.

Reply:
Thanks for explaining, fine for me